connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -169,6 +169,14 @@
      */
     void resumeConnector(String connector);
 
+    /**
+     * Goes through config parameters and replace password field value with "[hidden"]
+     * @param connName name of the connector
+     * @param config configuration of the connector
```

Comment:
I'd suggest adding `; may not be null` to both of these `@param` lines. Also, might want to clarify that password fields are those defined with the type {@link ConfigDef.Type#PASSWORD} in the `ConfigDef`.

Reply:
Sure, I'll add that in.